components/cdc/src/initializer.rs

```diff
@@ -224,6 +232,28 @@ impl<E: KvEngine> Initializer<E> {
             Err(box_err!("scan canceled"))
         };
 
+        let mut scanner_iter: Option<ScannerIter<S>>;
+        if !(kv_api == ChangeDataRequestKvApi::RawKv) {
```

Comment:
Suggest limit to `TiDB` now, before we implement CDC for TxnKV. To avoid some unexpected behaviors.